### PR TITLE
ENCD-4722 install Valis as component

### PR DESCRIPTION
**Pull request by emro**
We want to load the Valis genome visualizer as a component.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4136,20 +4603,17 @@
...
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "console-control-strings": {
           "version": "1.1.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
```
Comment (emro) [inline package-lock.json]: I can go through and undo these changes if they're not desired.

File: webpack.config.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ const loaders = [
             path.resolve(__dirname, 'node_modules/superagent'),
         ],
         loader: 'babel',
+        query: { compact: false },
```
Comment (emro) [inline webpack.config.js]: [Here](https://stackoverflow.com/questions/29576341/what-does-the-code-generator-has-deoptimised-the-styling-of-some-file-as-it-e) is why I added this.

File: package-lock.json (hunk 1 above)
Comment (forresttanaka) [reply package-lock.json]: My impression right now is that this is just the new way of things in npm world.

Comment (forresttanaka) [reply package-lock.json]: 👍

File: webpack.config.js (hunk 2 above)
Comment (forresttanaka) [reply webpack.config.js]: 👍

**Comment by emro** [comment]
@forresttanaka I think you approved this because it was in QA Ready but somehow Github didn't have it labeled as approved.

**Comment by forresttanaka** [comment]
I remember reviewing this. Maybe I forgot to approve it at that time? I’m not sure.

**Pull request merged**